sdcm/cluster.py

```diff
@@ -3361,6 +3361,28 @@ def get_nodetool_info(node):
         info_res = yaml.load(proper_yaml_output)
         return info_res
 
+    @staticmethod
+    def get_node_id(node):
```

Comment:
why not using `get_nodetool_status` ? that already parses this ?

https://github.com/scylladb/scylla-cluster-tests/blob/25493b0a8d4011239acf888958c27ea1934fbc37/sdcm/cluster.py#L3326

Reply:
also it called `host id` and not `node_id`